topics/transcriptomics/tutorials/differential-isoform-expression/tutorial.md

```diff
@@ -744,20 +834,33 @@ We will try to confirm this {IS} by visualising the coverage on this gene and mo
 >        - {% icon param-repeat %} *"Insert Include tracks in your plot"*
 >            - *"Choose style of the track"*: `Gene track / Bed track`
 >                - *"Plot title"*: `Genes`
->                - {% icon param-file %} *"Track file(s) bed or gtf format"*: Select `gencode.v43.annotation.gtf.gz`
+>                - {% icon param-file %} *"Track file(s) bed or gtf format"*: Select `StringTie annotation`
```

Comment:
Do you think this is fine @lldelisle? By using the annotation generated by StringTie, it allows to visualize the new identified isoforms.

Reply:
Here you can see the differences https://github.com/galaxyproject/training-material/pull/4178/files#diff-99eca97f5c256615d6b5af5677344ceb285456b9323233fcb09df0c27c5ffcad